community/kernel/src/main/java/org/neo4j/kernel/impl/proc/MethodSignatureCompiler.java

```diff
@@ -62,6 +62,14 @@ public MethodSignatureCompiler( TypeMappers typeMappers )
             }
             String name = param.getAnnotation( Name.class ).value();
 
+            if( name.trim().length() == 0 )
+            {
+                throw new ProcedureException( Status.Procedure.FailedRegistration,
+                        "Argument at position %d in method `%s` is annotated with a name, but the name is empty, " +
```

Comment:
In printed form, this line will printed as one long line. Perhaps a literal newline in there somewhere?